js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -785,7 +786,7 @@ var DropDownEditor = TextBox.inherit({
     _updatePopupWidth: commonUtils.noop,
 
     _popupOptionChanged: function(args) {
-        var options = this._getOptionsFromContainer(args);
+        var options = Widget.getOptionsFromContainer(args);
```

Comment:
Should we use const/let instead of var?

Reply:
We should, but this file not refactored yet.